### PR TITLE
feat(onboarding): Update onboarding to match designs

### PR DESCRIPTION
**Pull request by jophish**
### Description
For [ACT-1415](https://linear.app/valora/issue/ACT-1415/mento-app-rest-of-the-onboarding-flow). Updates the onboarding flow to match designs. Since we still don't have access to the Google developer account we'll use for Mento, CAB isn't fully set up for Android, though it should work for iOS at the moment. Once we get access to the dev account, we'll need to update some config both in this repo as well in the backend to get CAB working correctly for Android.

### Test plan

Manual testing; everything is just style changes

### Related issues

- Fixes [ACT-1415](https://linear.app/valora/issue/ACT-1415/mento-app-rest-of-the-onboarding-flow)

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [ ] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: locales/base/translation.json [hunk 1]
```diff
@@ -842,7 +842,7 @@
     "titleGuided": "Welcome to {{appName}}!\nLet’s create your crypto wallet.",
     "createAccount": "Create new account",
     "restoreAccount": "Restore my account",
-    "createNewWallet": "Create new wallet",
+    "createNewWallet": "Create new account",
```
Comment (satish-ravi) [inline locales/base/translation.json]: I think we should hold off on changing wallet -> account, or if we intend to do this, we should also change the hasWallet copy? Last I checked with product, this wasn't confirmed

File: locales/base/translation.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
   },
   "signInWithEmail": {
     "title": "Sign in with email",
-    "subtitle": "Sign in with your email to set it up as a backup. You will be prompted to sign in again if you ever lose your wallet.",
+    "subtitle": "Remember this email address & password. You will be prompted to sign in again if you ever lose your wallet.",
```
Comment (satish-ravi) [inline locales/base/translation.json]: Looks like we have different copies for CAB setup in onboarding vs settings per figma, I wonder if this was a bug in Valora (i.e., when we added CAB setup in onboarding we never did the different copies for this screen). Maybe we should fix this in Valora?

File: locales/base/translation.json [hunk 3]
```diff
@@ -77,7 +77,7 @@
     "title": "Your recovery phrase",
     "titleV1_90": "Your Recovery Phrase",
     "body": "Write these 12 words down somewhere safe, where it cannot be stolen or damaged.",
-    "bodyV1_90": "If you lose your recovery phrase, you will lose access to all funds held in {{appName}}.",
+    "bodyV1_90": "If you lose your recovery phrase, you will lose access to all of your assets on {{appName}}. Nobody, not even {{appName}} will be able to help you recover your account without your Recovery Phrase. Store it in a safe place and do not share it.",
```
Comment (satish-ravi) [inline locales/base/translation.json:80]: Is this a mento specific change? Wonder if it was design being out of date w/ actual implementation (applies to other copy changes)

File: locales/base/translation.json (hunk 1 above)
Comment (jophish) [reply locales/base/translation.json]: Yeah, will revert this for the time being.

File: locales/base/translation.json (hunk 2 above)
Comment (jophish) [reply locales/base/translation.json]: Hmm, I asked Kayla about this, will leave this as-is till I hear back.

File: locales/base/translation.json (hunk 3 above)
Comment (jophish) [reply locales/base/translation.json:80]: Also asked Kayla about this!

Comment (jophish) [reply locales/base/translation.json:80]: I think this change is intended, as it matches the existing `postSetupBody` translation key.

Comment (satish-ravi) [reply locales/base/translation.json:80]: is this something that should've been done on Valora then?

File: src/components/CodeInput.tsx [hunk 4]
```diff
@@ -103,9 +103,8 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: Spacing.Regular16,
     overflow: 'hidden',
-    backgroundColor: colors.gray1,
     borderWidth: 1,
-    borderColor: colors.gray2,
+    borderColor: colors.black,
```
Comment (satish-ravi) [inline src/components/CodeInput.tsx:107]: wouldn't it be nice if we had a shared component for this? I've been making these changes everywhere too 😄

File: src/keylessBackup/KeylessBackupPhoneInput.tsx [hunk 5]
```diff
@@ -197,7 +197,8 @@ const styles = StyleSheet.create({
     padding: Spacing.Thick24,
   },
   countryFlagStyle: {
-    backgroundColor: Colors.gray2,
-    marginRight: Spacing.Smallest8,
```
Comment (satish-ravi) [inline src/keylessBackup/KeylessBackupPhoneInput.tsx:201]: I saw a change where a margin on phone number input changed from 16->24, is that just being made up here? If so, can we just leave it as is (to keep diffs from fork minimal)

**Review by satish-ravi — APPROVED** [review]

File: src/keylessBackup/KeylessBackupPhoneInput.tsx (hunk 5 above)
Comment (jophish) [reply src/keylessBackup/KeylessBackupPhoneInput.tsx:201]: Without this rejiggering, the on-press shadow on the country selector doesn't respect the border radii on the right side and renders with right angles, strangely enough.

**Pull request merged**